client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -293,7 +293,14 @@ const LowerJaw = ({
       : t('buttons.check-code')
     : t('buttons.check-code-2');
 
+  const submitButtonText = isDesktop
+    ? isMacOS
+      ? t('buttons.submit-and-go-3')
+      : t('buttons.submit-and-go')
+    : t('buttons.submit-and-go-2');
```

Comment:
```suggestion
      : t('buttons.submit-and-go-2')
    : t('buttons.submit-and-go');
```

Sorry, the properties are apparently switched for the submit button text.

I might open a PR after this is merged to give them better names and make sure the default is consistent.